### PR TITLE
Make it compatible with current eris dev branch

### PR DESCRIPTION
**Pull request by jtsshieh**
pls give me free pr kthx

### CODE REVIEW

File: lib/ReactionHandler.js [hunk 1]
```diff
@@ -29,16 +29,16 @@ class ReactionHandler extends EventEmitter {
      * Verify a reaction for its validity with provided filters
      * @param {object} msg The message object 
      * @param {object} emoji The emoji object containing its name and its ID 
-     * @param {string} userID The user ID of the user who's reacted 
+     * @param {Member} reactor The member who reacted to this message
```
Comment (knht) [inline lib/ReactionHandler.js]: I'd prefer if you made clear it's an Eris Member object:

`* @param {Eris.Member} reactor ...`

**Review by knht — CHANGES_REQUESTED** [review]
Also please update the README file with the updated API as it's not auto generated from JSDoc but manually written.

**Comment by jtsshieh** [comment]
I don't think the API was changed at all. This function doesn't seem to be documented on the README and the ID is still passed into the filter and the user. It's just set up to receive member objects from Eris.

**Review by knht — APPROVED** [review]

**Comment by knht** [comment]
Will publish to NPM once Eris 0.14.0 is released :)

**Pull request merged**